netutils/lib_mapper.py

```diff
@@ -609,7 +611,9 @@
 }
 
 # Normalized | NIST
+# TODO: This should be deprecated in next major release
```

Comment:
I think you can remove this, we would keep this regardless of `NIST_LIB_MAPPER_REVERSE_DICT`. This is the standard to map between all of the different OS's.

Reply:
Removed.

Reply:
I also updated the NIST_LIB_MAPPER_REVERSE to keep consistent entries with the others since the mapper itself is being kept for standard.